amundsen_application/api/mail/v0.py

```diff
@@ -45,7 +52,8 @@ def feedback() -> Response:
                   repro_steps=repro_steps,
                   feature_summary=feature_summary,
                   value_prop=value_prop,
-                  subject=subject)
+                  subject=subject,
+                  user_id=user_id)
```

Comment:
There should be a way to capture the user in the action log without having to add this in here, we have that working internally at Lyft. I think we have to chance the following line of code to use `user_id` and not `email`
https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/log/action_log.py#L82

At Lyft we currently use the `email` as the `user_id` and must have missed updating that one line.